cli/ballerina-cli-module-pull/build.gradle

```diff
@@ -64,10 +64,10 @@ dependencies {
     baloImplementation project(path: ':ballerina-stringutils', configuration: 'baloImplementation')
     baloImplementation project(path: ':ballerina-java', configuration: 'baloImplementation')
 
-    interopImports project(':ballerina-filepath')
```

Comment:
Why was this removed?

Reply:
I have added. https://github.com/ballerina-platform/ballerina-lang/pull/20095/files#diff-69d9c615e318006f2090fba78b10148bR68

Reply:
Ack